runtime/flang/heapinit.c

```diff
@@ -33,10 +33,14 @@ int val;
   void (*save)();
   int *pi;
 
+#ifndef _WIN64
```

Comment:
What will happen in windows?

Reply:
We don't need to handle the signal on windows, since it doesn't exist.